pages/common/qpdf.md

```diff
@@ -11,6 +11,10 @@
 
```

Comment:
Now this description becomes slightly misleading because of the new one. We should change it to - "Merge (concatenate) given pages from a list of PDF files and save the result as a new PDF"